packages/devtools_test/lib/src/mocks/generated.mocks.dart

```diff
@@ -0,0 +1,725 @@
+// Mocks generated by Mockito 5.0.17 from annotations
```

Comment:
add copyright

Reply:
hm. it is generated file. So, we will need to re-add it every time after generation.

Reply:
while a bit tedious, we'll need to to follow the open source policy guidelines